app/controllers/users.js

```diff
@@ -34,3 +34,17 @@ exports.signIn = async (req, res, next) => {
     next(error);
   }
 };
+
+exports.getUsers = async (req, res, next) => {
+  try {
+    const page = req.query.page ? req.query.page - 1 : 0;
+    const limit = req.query.limit || 10;
+    const offset = limit * page;
+    const users = await getUsers(limit, offset);
+    if (!users) throw notFoundError(userNotFoundErrorMessage);
```

Comment:
This logic would be better in an Interactor. An interactor works in the same layer than a controller, implementing reusable business logic, leaving the controller as clean as possible.
Also, the pagination logic could be abstracted.

Reply:
Okay, I already transferred all the logic to the interactors